### PR TITLE
Add kafka_exporter, a tool for sending Kafka stats to Prometheus

### PR DESCRIPTION
**Pull request by nightscape**

### CODE REVIEW

File: bucket/kafka_exporter.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.2.0",
+    "homepage": "https://github.com/danielqsj/kafka_exporter",
+    "description": "Kafka exporter for Prometheus",
+    "bin": "kafka_exporter.exe",
+    "checkver": {
+        "github": "https://github.com/danielqsj/kafka_exporter"
```
Comment (niheaven) [inline bucket/kafka_exporter.json]:
```json
"checkver": "github"
```

This is enough.

Comment (nightscape) [reply bucket/kafka_exporter.json]: Amended, thanks for the tip!

**Comment by nightscape** [comment]
@niheaven I saw you are moving command-line-only programs to the main bucket.
Do programs like this that expose something over http/tcp/... count as command-line programs as well?

**Comment by r15ch13** [comment]
Please change the filename to `kafka-exporter.json`.
If you run it from the CLI and it doesn't show a Windows GUI then it's better suited for the Main bucket.

File: bucket/kafka_exporter.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "extract_dir": "kafka_exporter-1.2.0.windows-amd64",
+            "url": "https://github.com/danielqsj/kafka_exporter/releases/download/v1.2.0/kafka_exporter-1.2.0.windows-amd64.tar.gz",
+            "hash": "c994cb8009d3fdf78589c507eaccf750a69664f304192e094be2762bc999db76"
+        }
+    },
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "extract_dir": "kafka_exporter-$version.windows-386",
+                "url": "https://github.com/danielqsj/kafka_exporter/releases/download/v1.2.0/kafka_exporter-1.2.0.windows-386.tar.gz"
```
Comment (Ash258) [inline bucket/kafka_exporter.json]: Prefilled autoupdate URLs.

Comment (nightscape) [reply bucket/kafka_exporter.json]: Good catch!

**Comment by nightscape** [comment]
Moved to https://github.com/ScoopInstaller/Main/pull/125

**Pull request closed without merging**